src/clusterStatusCheck.ts

```diff
@@ -1,10 +1,10 @@
 import {ssmCommand} from './utils/ssmCommand';
 import {StandardOutputContent} from 'aws-sdk/clients/ssm';
 
-export async function handler(instanceId: string): Promise<string> {
+export async function handler(instanceId: string): Promise<Object> {
     return ssmCommand('curl localhost:9200/_cluster/health', instanceId)
         .then((result: StandardOutputContent) => {
             const json = JSON.parse(result);
-            return json.status;
+            return { "instanceId": instanceId, "clusterStatus": json.status } ;
```

Comment:
The next lambda (after the choice) will need the instanceId, so we now pass this through here.